processor/schemaprocessor/processor.go

```diff
@@ -48,6 +48,44 @@ func newSchemaProcessor(_ context.Context, conf component.Config, set processor.
 }
 
 func (t schemaProcessor) processLogs(_ context.Context, ld plog.Logs) (plog.Logs, error) {
+	for rt := 0; rt < ld.ResourceLogs().Len(); rt++ {
+		rLogs := ld.ResourceLogs().At(rt)
+		resourceSchemaURL := rLogs.SchemaUrl()
+		if resourceSchemaURL != "" {
+			t.log.Debug("requesting translation for resourceSchemaURL", zap.String("resourceSchemaURL", resourceSchemaURL))
+			tr, err := t.manager.
+				RequestTranslation(context.Background(), resourceSchemaURL)
```

Comment:
Can I ask why is this using the background context instead of the context passed via the method?

Reply:
it was miss. Thanks for catching it.